### PR TITLE
fixing vulnerability

### LINKED ISSUE #45: xss vulnerability in search results page

### PR DESCRIPTION
**Pull request by carl-alberto**
closes #45

### CODE REVIEW

**Comment by wpugphil** [comment]
**PHP Syntax Errors Found**


Scan performed on the code at commit 4f2ea209c8c405f9107a5d3b3fd05e709b8d7cd1 ([view code](https://github.com/wpugph/dctx-website/tree/4f2ea209c8c405f9107a5d3b3fd05e709b8d7cd1)).

***

**Error**: syntax error, unexpected `)`, expecting `,` or `;`


https://github.com/wpugph/dctx-website/blob/4f2ea209c8c405f9107a5d3b3fd05e709b8d7cd1/web/wp-content/themes/dctx/search.php#L16

File: web/wp-content/themes/dctx/search.php [hunk 1]
```diff
@@ -8,20 +8,21 @@
...
 
-// phpcs:disable
-
 get_header();
+$search_result = get_query_var( 's', 1 );
 ?>
 
 <div id="main">
-	<h1 class="page-title">Search results for '<?php echo $_GET['s']; ?>'</h1>
+	<h1 class="page-title">Search results for '<?php echo $search_result ); ?>'</h1>
```
Comment (wpugphil) [inline web/wp-content/themes/dctx/search.php:16]: :no_entry_sign: **Error**: All output should be run through an escaping function (see the Security sections in the WordPress Developer Handbooks), found '$search_result' (*WordPress.Security.EscapeOutput.OutputNotEscaped*).

Comment (wpugphil) [inline web/wp-content/themes/dctx/search.php:16]: :no_entry_sign: **Error**: PHP syntax error: syntax error, unexpected ')', expecting ',' or ';' (*Generic.PHP.Syntax.PHPSyntax*).

**Comment by carl-alberto** [comment]
This should fix security fix raised here https://dctx.slack.com/archives/G010EBC65T3/p1585989567001100

**Pull request merged**